webhook/server/server.go

```diff
@@ -75,10 +77,16 @@ func registerInjectEndpoint(mgr manager.Manager, ns string, podName string) erro
 		return err
 	}
 
+	var clusterNS corev1.Namespace
+	if err := mgr.GetAPIReader().Get(context.TODO(), client.ObjectKey{Name: "kube-system"}, &clusterNS); err != nil {
+		return err
+	}
+
```

Comment:
A function which returns the kube-system UID  already exists in controllers/kubesystem.
I suggest using it to avoid code duplication

Reply:
Ok, thx